components/sst_importer/src/sst_importer.rs

```diff
@@ -508,7 +508,9 @@ impl<E: KvEngine> SstImporter<E> {
             })?;
         }
 
-        let ext_storage = self.external_storage_or_cache(backend, cache_key)?;
+        let ext_storage = tokio::task::block_in_place(move || {
```

Comment:
Would you also add some comments that explain why this `block_in_place` is necessary?